core/src/main/scala/kafka/controller/PartitionLeaderSelector.scala

```diff
@@ -134,9 +133,10 @@ class ReassignedPartitionLeaderSelector(controllerContext: ControllerContext) ex
  * New isr = current isr;
  * Replicas to receive LeaderAndIsr request = assigned replicas
  */
-class PreferredReplicaPartitionLeaderSelector(controllerContext: ControllerContext) extends PartitionLeaderSelector
-with Logging {
-  this.logIdent = "[PreferredReplicaPartitionLeaderSelector]: "
+class PreferredReplicaPartitionLeaderSelector(controllerContext: ControllerContext)
+  extends PartitionLeaderSelector
+    with Logging {
```

Comment:
These things are a bit subjective though. Especially when there are only 2 classes/traits listed. I suggest sticking to the style used by the project as a general rule.